cmake/functions.cmake

```diff
@@ -61,7 +61,7 @@ function(compile_proto_to_cpp PROTO_LIBRARY_NAME PB_H PB_CC PROTO)
     message(FATAL_ERROR "Protobuf_INCLUDE_DIR is empty")
   endif ()
 
-  get_filename_component(PROTO_ABS "${PROTO}" REALPATH)
+  get_filename_component(PROTO_ABS "${PROTO}" ABSOLUTE)
```

Comment:
🤷‍♂️

Reply:
`REALPATH` resolves symlinks, `ABSOLUTE` resolves `..`.
When `.hunter` was symlink (due to disk/partition size), protobuf generator complained about paths (it doesn't resolve symlinks).